api/handlers_test.go

```diff
@@ -250,3 +252,212 @@ func TestObjectLifeline_Index_Error(t *testing.T) {
+		jetDrop3.JetID = jetID3.Prefix()
+		err = testutils.CreateJetDrop(testDB, jetDrop3)
+		require.NoError(t, err)
+
+		jetDrop4 := testutils.InitJetDropDB(pulse)
+		jetID4 := jet.NewIDFromString("011")
+		jetDrop4.JetID = jetID4.Prefix()
+		err = testutils.CreateJetDrop(testDB, jetDrop4)
+		require.NoError(t, err)
+
+		jetDrop5 := testutils.InitJetDropDB(pulse)
+		jetID5 := jet.NewIDFromString("100")
+		jetDrop5.JetID = jetID5.Prefix()
+		err = testutils.CreateJetDrop(testDB, jetDrop5)
+		require.NoError(t, err)
+
+		jetDrop6 := testutils.InitJetDropDB(pulse)
+		jetID6 := jet.NewIDFromString("101")
+		jetDrop6.JetID = jetID6.Prefix()
+		err = testutils.CreateJetDrop(testDB, jetDrop6)
+		require.NoError(t, err)
+
+		resp, err := http.Get(
+			"http://" + apihost + "/api/v1/pulses/" +
+				strconv.Itoa(pulse.PulseNumber) +
+				"/jet-drops?limit=2&offset=2&from_jet_drop_id=" +
+				jetDropID2,
+		)
+		require.NoError(t, err)
+		require.Equal(t, http.StatusOK, resp.StatusCode)
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
+		require.NoError(t, err)
+
+		var received server.JetDropsResponse
+		err = json.Unmarshal(bodyBytes, &received)
+		require.NoError(t, err)
+		require.EqualValues(t, 2, int(*received.Total))
+		require.Len(t, *received.Result, 2)
+		// check asc order by default
+		require.Equal(t, models.JetDropID(jetDrop5.JetID, int64(pulse.PulseNumber)), *(*received.Result)[0].JetDropId)
+		require.Equal(t, models.JetDropID(jetDrop6.JetID, int64(pulse.PulseNumber)), *(*received.Result)[1].JetDropId)
+	})
+
+	t.Run("error wrong jetdropid", func(t *testing.T) {
+		pulse, err := testutils.InitPulseDB()
+		require.NoError(t, err)
+		resp, err := http.Get(
+			"http://" + apihost + "/api/v1/pulses/" +
+				strconv.Itoa(pulse.PulseNumber) +
+				"/jet-drops?from_jet_drop_id=" +
+				"test",
+		)
+		require.NoError(t, err)
+		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
+
+		resp, err = http.Get(
+			"http://" + apihost + "/api/v1/pulses/" +
+				strconv.Itoa(pulse.PulseNumber) +
+				"/jet-drops?from_jet_drop_id=" +
+				"10076767676",
+		)
+		require.NoError(t, err)
+		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
+
+		resp, err = http.Get(
+			"http://" + apihost + "/api/v1/pulses/" +
+				strconv.Itoa(pulse.PulseNumber) +
+				"/jet-drops?from_jet_drop_id=" +
+				"76767676:1000",
+		)
+		require.NoError(t, err)
+		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
+	})
+
+	t.Run("error wrong pulse", func(t *testing.T) {
+		resp, err := http.Get(
+			"http://" + apihost + "/api/v1/pulses/" +
+				"wrong-pulse" +
+				"/jet-drops",
+		)
+		require.NoError(t, err)
+		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
+
+	})
+
+	t.Run("error wrong limit", func(t *testing.T) {
+		resp, err := http.Get(
+			"http://" + apihost + "/api/v1/pulses/" +
+				"wrong-pulse" +
+				"/jet-drops?limit=" + "we248934h9h'`;",
+		)
+		require.NoError(t, err)
+		require.Equal(t, http.StatusBadRequest, resp.StatusCode)
+
+	})
```

Comment:
Add test for case when we ask for jetdrops from not-existing pulse

Reply:
yep